runners/direct-java/build.gradle

```diff
@@ -57,12 +57,14 @@ dependencies {
   compile project(path: ":beam-runners-core-java", configuration: "shadow")
   compile project(path: ":beam-runners-local-java-core", configuration: "shadow")
   compile project(path: ":beam-runners-java-fn-execution", configuration: "shadow")
+  compile project(path: ":beam-sdks-java-fn-execution", configuration: "shadow")
   shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
   shadow library.java.joda_time
   shadow library.java.findbugs_jsr305
   shadow library.java.slf4j_api
   shadow library.java.hamcrest_core
   shadow library.java.junit
+  shadow library.java.args4j
```

Comment:
should we remove args4j from reference runner as a dependency?

Reply:
It's not a dependency of that module.

Reply:
It was the job server that had it which you deleted below.